modules/mint-mode-with-removal-of-admin-credential.adoc

```diff
@@ -3,25 +3,19 @@
 // * installing/installing_aws/manually-creating-iam.adoc
 
 [id="mint-mode-with-removal-or-rotation-of-admin-credential_{context}"]
-= Mint Mode with removal or rotation of the admin credential
+= Mint mode with removal or rotation of the administrator-level credential
```

Comment:
Should this mode be in the mode list in [Modes](https://github.com/openshift/openshift-docs/pull/29545/files#diff-76786602ed2bd71315fdf4b992f73b242cf54a482cf18d893838489034b7405eR14)?

Reply:
It's not really a separate mode, but I suppose STS isn't either. I'll think about this :)

Reply:
Tracking in [OSDOCS-1817](https://issues.redhat.com/browse/OSDOCS-1817)